Dockerfile

```diff
@@ -38,35 +47,8 @@ RUN apt-get update && \
     # best practice to keep the Docker image lean
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install essential Python packages
-RUN pip3 --no-cache-dir install \
-    numpy \
-    matplotlib \
```

Comment:
So in this Docker image, we are not going to put any additional scientific packages, are we? I guess we'll put it in a downstream image?

Reply:
Ah yeah I need to remove that, missed that